packages/devextreme/js/__internal/scheduler/appointment_popup/m_popup.ts

```diff
@@ -473,6 +481,9 @@ export class AppointmentPopup {
       shortcut: 'cancel',
       options: {
         stylingMode: 'outlined',
+        onClick: (): void => {
+          this.hide();
+        },
```

Comment:
Looks like we don't need this handler. Property `shortcutL 'cancel'` already should tell popup to close itself when this button is clicked.

I have tested without this function and onCancel callback was still called